### PR TITLE
Restructure the Symbol Library chapter

### PR DESCRIPTION
**Pull request by DelazJ**
Move the Color ramp section under Style manager chapter

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/style_library.rst [hunk 1]
```diff
@@ -91,7 +91,7 @@ different tabs:
 * **Marker** for point symbols
 * **Line** for linear symbols
 * **Fill** for surface symbols
-* `Color Ramp`_
+* and `Color Ramp`_
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/style_library.rst]: need word in bold, isn't?

**Comment by yjacolin** [comment]
looks good to me

File: source/docs/user_manual/working_with_vector/style_library.rst (hunk 1 above)
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/style_library.rst]: It's rather an hyperlink. Should I bold it also? If so, how? I tried **`Color Ramp`_** and `**Color Ramp**`_ unsurprisingly without success

Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/style_library.rst]: never mind, go ahead! RST is not perfect, I can live without :)

**Pull request merged**